docs/conf.py

```diff
@@ -42,8 +42,15 @@
     "sphinx.ext.intersphinx",
     "sphinx.ext.mathjax",
     "sphinx.ext.viewcode",
+    "jupyter_sphinx",
```

Comment:
I have some suggestions on the `jupyter_sphinx` usage. I can discuss details with you in person sometime.

The quantinuum docs project uses [MyST-NB](https://myst-nb.readthedocs.io/en/latest/) which has similar functionality but is my preferred library for this sort of thing.

Reply:
This tooling is interchangeable to some extent

Reply:
Thanks for drawing my attention to that! I've had a play around with it and like it very much. In general I think I will prefer myst to rst. I have much to learn but I'm optimistic it will be good.
Thanks!